accounting/eval_preload.go

```diff
@@ -22,7 +22,12 @@ func addToCreatorsRequest(stxnad *transactions.SignedTxnWithAD, assetsReq map[ba
 		}
 	case protocol.AssetTransferTx:
 		fields := &txn.AssetTransferTxnFields
-		if fields.XferAsset != 0 {
+
+		noOpXfer := (fields.AssetAmount == 0) &&
```

Comment:
Right - since we certainly grabbed the asset to check the clawback addr, even if the movement is 0 amount (I check, as I suppose we don't _have_ to check a 0 clawback)